grafana: Add secret_env variable (#541)
kinvolk/lokomotive: docs/configuration-reference/components/prometheus-operator.md

  grafana {
    admin_password = "foobar"

>     secret_env     = {            # Declare variable to define this parameter and define it in `lokocfg.vars`.

Reviewer: ```suggestion
    secret_env     = {            # This might contain sensitive information, declare a variable and define this in `lokocfg.vars`.
```

>     secret_env     = {        # This might contain sensitive information, declare a variable and define this in `lokocfg.vars`.

Reviewer 2: nit: terraform fmt

Author: done.

|--------	|--------------|:-------:|:--------:|
| `namespace` | Namespace to deploy the Prometheus Operator. | `monitoring` | false |
| `grafana.admin_password` | Password for `admin` user in Grafana. If not provided it is auto generated and stored in secret `prometheus-operator-grafana`.  | - | false |

> | `grafana.env_from_secret` | Name of the Secret to populate environment variables in the container. The contents of the target Secret's Data field will represent the key-value pairs as environment variables.  | - | false |

Reviewer 3: Could you please fix the caps here?

|--------	|--------------|:-------:|:--------:|
| `namespace` | Namespace to deploy the Prometheus Operator. | `monitoring` | false |
| `grafana.admin_password` | Password for `admin` user in Grafana. If not provided it is auto generated and stored in secret `prometheus-operator-grafana`.  | - | false |

> | `grafana.secret_env` | Sensitive environment variables passed to Grafana pod and stored as secret. | - | false |

Reviewer: Maybe link to https://grafana.com/docs/grafana/latest/installation/configuration/#configure-with-environment-variables?

Author: Added additional line. Even though this will be covered in a doc.
